homeassistant/components/nuki/lock.py

```diff
@@ -109,23 +123,40 @@ def device_state_attributes(self):
         """Return the device specific state attributes."""
         data = {
             ATTR_BATTERY_CRITICAL: self._battery_critical,
-            ATTR_NUKI_ID: self._nuki_lock.nuki_id}
+            ATTR_NUKI_ID: self._nuki_lock.nuki_id,
+            ATTR_LOCK_REACHABLE: self._lock_reachable}
         return data
 
+    @property
+    def supported_features(self):
+        """Flag supported features."""
+        return SUPPORT_OPEN
+
     def update(self):
         """Update the nuki lock properties."""
-        self._nuki_lock.update(aggressive=False)
-        self._name = self._nuki_lock.name
-        self._locked = self._nuki_lock.is_locked
-        self._battery_critical = self._nuki_lock.battery_critical
+        try:
+            self._nuki_lock.update(aggressive=False)
+        except requests.exceptions.RequestException:
+            self._lock_reachable = False
```

Comment:
Add the `available` property and return this value.

Reply:
You mean change `lock_reachable` to `available`?

Reply:
yes

Reply:
Done